File: packages/runtime-java/src/main/java/besdk/runtime/ValidationMode.java
package besdk.runtime;

/** How strictly a response is checked against the shape the spec declared (SPEC.md §3.4.1.1). */
public enum ValidationMode {
  /** Throw a {@link ResponseValidationException} naming the offending field. The default. */
  STRICT,
  /** Log and continue. */
  WARN,
  /** Skip the check. */
  OFF
}

File: packages/runtime-php/src/ValidationMode.php
<?php

declare(strict_types=1);

namespace Besdk\Runtime;

/** How strictly a response is checked against the shape the spec declared (SPEC.md §3.4.1.1). */
enum ValidationMode: string
{
    case Strict = 'strict';
    case Warn = 'warn';
    case Off = 'off';
}
